packages/cache-inmemory/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@medusajs/cache-inmemory",
+  "version": "1.0.0",
+  "description": "In-memory Cache Module for Medusa",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/cache-inmemory"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Medusa",
+  "license": "MIT",
+  "devDependencies": {
+    "@medusajs/medusa": "*",
+    "cross-env": "^5.2.1",
+    "jest": "^25.5.2",
+    "ts-jest": "^25.5.1",
+    "typescript": "^4.4.4"
+  },
+  "scripts": {
+    "watch": "tsc --build --watch",
+    "prepare": "cross-env NODE_ENV=production yarn run build",
+    "build": "tsc --build",
+    "test": "jest --passWithNoTests",
+    "test:unit": "jest --passWithNoTests"
+  },
+  "peerDependencies": {
+    "@medusajs/medusa": "1.7.5",
+    "medusa-interfaces": "1.3.4"
+  },
+  "dependencies": {
+    "typeorm": "^0.2.31"
```

Comment:
todo:I would keep this as a peer deps like the other packages. We did the same in the typeorm pr for the other modules